### DISCUSSION TITLE
How should we advertise Bark?

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#70] [Q&A] [answered]
All in the title

**Comment by nmsderp** [comment]
scratch forums

**Comment by raynepaws** [comment, marked as answer]
- as nmsderp has stated, scratch forums.
- i also have a social media account with 90 something followers and i could make a post or two there
- youtube

**Comment by mariocraft987** [comment]
nice

**Comment by rubiidev18alt** [comment]
not, but sorta related to the question: can I edit up a 3rd trailer for bark and let mariocraft upload it along myself?

**Comment by mariocraft987** [comment]
k
